CONTRIBUTING.md

```diff
@@ -21,11 +20,11 @@ The following instructions detail the process for claiming and completing tasks.
 
 - To claim a task, comment the single word `claim` on the relevant GitHub issue.
 - If no other user is assigned, you will automatically be assigned to the task, and the issue will move to the `Claimed` column.
-- You may only claim one task at a time. If you decide not to work on a task after claiming it, comment the single word `disclaim` on the issue. This will unassign you and return the issue to the `Unclaimed` column, making it available for others to claim.
+- If you decide not to work on a task after claiming it, comment the single word `disclaim` on the issue. This will unassign you and return the issue to the `Unclaimed` column, making it available for others to claim.
 
 ### 3. Working on the Task
 
-Once you are assigned to an issue, begin working on the corresponding task. You should create a new branch from the `main` branch to develop your solution.
+Once you are assigned to an issue, begin working on the corresponding task. You should fork the project and also create a new branch from the `main` branch to develop your solution. 
```

Comment:
```suggestion
Once you are assigned to an issue, begin working on the corresponding task. You should fork the project and also create a new branch from the `main` branch to develop your solution. Please try and avoid making PRs from `main` because apparently it confuses CI.
```